packages/cli/src/commands/deploy/aws.js

```diff
@@ -1,111 +1,35 @@
-import fs from 'fs'
-import path from 'path'
-
-import execa from 'execa'
-import Listr from 'listr'
-import VerboseRenderer from 'listr-verbose-renderer'
 import terminalLink from 'terminal-link'
 
-import { getPaths } from '../../lib'
-import c from '../../lib/colors'
-
-export const command = 'aws [provider]'
+export const command = 'aws <provider>'
```

Comment:
maybe we can rename this to serverless, and add an alias ```export const aliases = ['aws serverless']``` for backwards compatibility

Reply:
DO IT

Reply:
yeah just like the snippet :)

Reply:
Do we need this? There is no `aws-providers` subdirectory...I'm not sure what the point of this was:

```
    .commandDir('./aws-providers', {
      recurse: true,
      /*
    @NOTE This regex will ignore all double nested commands
    e.g. /setup/hi.js & setup/hi/hi.js are picked up,
    but setup/hi/hello/bazinga.js will be ignored
    The [\/\\] bit is for supporting both windows and unix style paths
    */
      exclude: /aws-providers[\/\\]+.*[\/\\]+.*[\/\\]/,
    })
```

Reply:
Okay see how that looks...instead of having the separate one in `aws-providers` I just combined them into the top level like Netlify and Vercel.

Reply:
We don't actually do anything with the provider so I saw we just leave that param off for now since it has no effect.